packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -78,6 +81,11 @@ export const useResizeHandle = (params: UseResizeHandleParams) => {
   const elementRef = React.useRef<HTMLElement | null>(null);
 
   const currentValue = React.useRef(UNMEASURED);
+  const handleChange: UseResizeHandleParams['onChange'] = useEventCallback(
+    (value) => {
+      onChange?.(value);
+    }
+  );
```

Comment:
We need to use `useEventCallback()` callback there, otherwise we make memoization an implicit contract